app/Console/Commands/DevicePoll.php

```diff
@@ -27,69 +24,19 @@ public function __construct()
         $this->addOption('no-data', 'x', InputOption::VALUE_NONE);
     }
 
-    public function handle(MeasurementManager $measurements): int
+    public function handle(): int
     {
-        $this->configureOutputOptions();
-
-        if ($this->option('no-data')) {
-            Config::set('rrd.enable', false);
-            Config::set('influxdb.enable', false);
-            Config::set('influxdbv2.enable', false);
-            Config::set('prometheus.enable', false);
-            Config::set('graphite.enable', false);
-        }
-
-        try {
-            /** @var \LibreNMS\Poller $poller */
-            $poller = app(Poller::class, ['device_spec' => $this->argument('device spec'), 'module_override' => explode(',', $this->option('modules') ?? '')]);
```

Comment:
These are the two lines that need to be moved into the job.  Rest should stay here.

Reply:
I don't see how the try/catch logic can be maintained if we don't keep all this code together.